srcs/SetServer.cpp

```diff
@@ -188,7 +189,7 @@ int Server::recieve_msg(int client_fd, int i)
 		setClientId(client_fd);
 		setMessage(buf);
 		std::cout << buf << std::endl;
-		if(parseMessage(client_fd))
+		if(parseMessage(client_fd) == -1)
```

Comment:
can we change this function name to something else to make it clear? The function is more about finding an excuting the correct command (parsing happens in message constructor). For example findCommand etc?